consensus/src/step_votes_reg.rs

```diff
@@ -96,35 +103,28 @@ pub type SafeCertificateInfoRegistry = Arc<Mutex<CertInfoRegistry>>;
 
 #[derive(Clone)]
 struct IterationCerts {
-    valid: Option<CertificateInfo>,
-    nil: CertificateInfo,
+    votes: HashMap<Vote, CertificateInfo>,
+    no_candidate: CertificateInfo,
```

Comment:
there's no point in having `no_candidate` now that we have the hashmap

Reply:
Updated